QuickOSM/core/process.py

```diff
@@ -165,6 +166,10 @@ def open_file(
                 QgsExpressionContextUtils.setLayerVariable(
                     new_layer, 'quickosm_query', final_query)
 
+            metadata = QgsLayerMetadata()
+            metadata.setRights(["OpenStreetMap"])
+            metadata.setLicenses(['https://osm.org/copyrights'])
```

Comment:
https://osm.org/copyrights is 404
Use https://www.openstreetmap.org/copyright

Reply:
It works without the final 's' dammit